build.gradle

```diff
@@ -336,8 +336,8 @@ dependencies {
 	
 	// https://mvnrepository.com/artifact/com.alphacephei/vosk
 	implementation group: 'com.alphacephei', name: 'vosk', version: '0.3.45'
-	
-
+	// Add OpenCV 
+	implementation 'org.openpnp:opencv:4.7.0-0'
```

Comment:
@JansenSmith changes to the build.gradle means you need to refresh the gradle, any time this file is changed in a PR, refresh the gradle.

Reply:
that was certainly an issue, thank you